scripts/debug_voq_chassis_packet_drops.sh

```diff
@@ -0,0 +1,371 @@
+#!/usr/bin/bash
```

Comment:
@vmittal-msft also if time permits, can you add a few comments on how to use this script ?